web/wp-content/themes/dctx2/inc/template-tags.php

```diff
@@ -41,38 +41,38 @@ function dctx_posted_on() {
 
 		echo '<span class="posted-on">' . $posted_on . '</span><span class="byline"> ' . $byline . '</span>'; // WPCS: XSS OK.
 
-	}
-endif;
-
-if ( ! function_exists( 'dctx_entry_footer' ) ) :
-	/**
-	 * Prints HTML with meta information for the categories, tags and comments.
-	 *
-	 * @author WDS
-	 */
-	function dctx_entry_footer() {
 		// Hide category and tag text for pages.
 		if ( 'post' === get_post_type() ) {
 			/* translators: used between list items, there is a space after the comma */
 			$categories_list = get_the_category_list( esc_html__( ', ', 'dctx' ) );
 			if ( $categories_list && dctx_categorized_blog() ) {
 				/* translators: the post category */
-				printf( '<span class="cat-links">' . esc_html__( 'Posted in %1$s', 'dctx' ) . '</span>', $categories_list ); // WPCS: XSS OK.
+				printf( '<span class="cat-links">' . esc_html__( ' under %1$s', 'dctx' ) . '.</span> ', $categories_list ); // WPCS: XSS OK.
 			}
 
 			/* translators: used between list items, there is a space after the comma */
 			$tags_list = get_the_tag_list( '', esc_html__( ', ', 'dctx' ) );
 			if ( $tags_list ) {
 				/* translators: the post tags */
-				printf( '<span class="tags-links">' . esc_html__( 'Tagged %1$s', 'dctx' ) . '</span>', $tags_list ); // WPCS: XSS OK.
+				printf( '<span class="tags-links">' . esc_html__( ' tagged %1$s', 'dctx' ) . '.</span> ', $tags_list ); // WPCS: XSS OK.
```

Comment:
:warning: **Warning**: Using the WPCS native whitelist comments is deprecated. Please use the PHPCS native &quot;phpcs:ignore Standard.Category.SniffName.ErrorCode&quot; annotations instead. Found: // WPCS: XSS OK.\n (*WordPress.Security.EscapeOutput.DeprecatedWhitelistCommentFound*).